templates/api/preprocessing/index.md

```diff
@@ -1,53 +0,0 @@
-# Dataset preprocessing
```

Comment:
It looks like the replacement of this file is missing from the PR?

Reply:
Oops forgot to stage it during the rename!